src/main/java/filemanager/launcher/Launcher.java

```diff
@@ -21,10 +22,11 @@ public static void main(String[] args) throws Exception {
     public void run(FileHandlerConfiguration configuration, Environment environment) {
         Injector guice = Guice.createInjector(new FileServiceBinderModule(configuration));
         ScheduleFileDirectory trackerFileDirectory = guice.getInstance(ScheduleFileDirectory.class);
+        Command command = guice.getInstance(Command.class);
         environment.jersey().register(guice.getInstance(ClientResource.class));
         environment.lifecycle().scheduledExecutorService("scheduledTracker")
                 .build()
-                .schedule((Runnable) trackerFileDirectory::goThroughToCheckFile, Long.parseLong(configuration.getTimeInterval()), TimeUnit.MINUTES);
+                .scheduleWithFixedDelay(() -> trackerFileDirectory.goThroughToCheckFile(command), 0, Long.parseLong(configuration.getTimeInterval()), TimeUnit.MINUTES);
```

Comment:
That's close. I bet that this code leads to reusing same time over and over again. So you aren't actually processing new folders with this code. Usage of Guice for Command creation is unnecessary here, just create new Command into this lambda directly. So there'll be a new time instance created every time this lambda gets executed.